addons/website/views/snippets.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 <template id="s_cover" name="Cover">
-    <section class="s_text_block_image_fw oe_img_bg oe_custom_bg" style="background-image: url(/web/image/website.s_cover_default_image); background-position: 50% 100%">
+    <section class="s_text_block_image_fw oe_img_bg" style="background-image: url(/web/image/website.s_background_image_03); background-position: 50% 100%">
```

Comment:
I understand but this fix is breaking a more "important" feature. If this really need to be solved, I think this can be done by changing the `set_active` function of the `background` snippet option to compare redirected URL or to compare real image content.